CONTRIBUTING.md

```
@@ -0,0 +1,74 @@
+cmake --build --preset dev
+```
+
+## Code style
+
+Formatting is defined by `.clang-format` and `.editorconfig`. Run clang-format on the code you change, and match the style of the surrounding code in matters the tools do not cover.
+
+## Commits
+
+Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/): a type, an optional scope, and a short description in the imperative mood.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)